packages/clarity-js/src/core/measure.ts

```diff
@@ -8,11 +8,11 @@ export default function(method: Function): Function {
         let start = performance.now();
         method.apply(this, arguments);
         let duration = performance.now() - start;
-        metric.accumulate(Metric.TotalDuration, duration);
+        metric.sum(Metric.TotalCost, duration);
         metric.count(Metric.InvokeCount);
```

Comment:
what are we trying to measure from invoke count? - if it is the number of times the clarity yielded, this shouldnt be counted here since measure is called for multiple function calls